test/utils.jl

```diff
@@ -16,11 +16,11 @@ module TestUtils
 
     # Check that reserved words are up to date
 
-    f = "$JULIA_HOME/../../src/julia-parser.scm"
+    f = "$Sys.BINDIR/../../src/julia-parser.scm"
```

Comment:
I think this needs to be `Compat.Sys.BINDIR`.

Reply:
Woops, due to the absence of parentheses this just interpolated `Sys`...